README.md

```diff
@@ -66,10 +66,10 @@ Then:
 
 ```bash
 # On Linux:
-labbox-launcher run magland/labbox-ephys:0.4.4 --docker_run_opts "--net host" --kachery $KACHERY_STORAGE_DIR
+labbox-launcher run magland/labbox-ephys:0.4.5 --docker_run_opts "--net host" --kachery $KACHERY_STORAGE_DIR
```

Comment:
Increment the version in all places where it is relevant. This is done each time we push a new image.